### PR TITLE
Customer/engagement meeting bug

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
Pull Request
**Are you a current Fivetran customer?** 
Fivetran created PR

**What change(s) does this PR introduce?** 
Changes mentions of variable `hubspot_engagement_email_enabled` to `hubspot_engagement_meeting_enabled` in:
- [models/stg_hubspot__engagement_meeting.sql](https://github.com/fivetran/dbt_hubspot_source/compare/customer/engagement-meeting-bug?expand=1#diff-11c7701611ee93171a48626eeb833b681f50cc4260fef22b9595d386b98b8599)
- [models/src_hubspot.yml](https://github.com/fivetran/dbt_hubspot_source/compare/customer/engagement-meeting-bug?expand=1#diff-96e62e4127b2948c9c874a9ca85222cc7fcc3e856c41153f04059c89f80b58ce)
- [models/tmp/stg_hubspot__engagement_meeting_tmp.sql](https://github.com/fivetran/dbt_hubspot_source/compare/customer/engagement-meeting-bug?expand=1#diff-ccdb00964453ee6d540dc33426ab299f690a18dafe6b85845843bb5ddc547442)

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature #84 
- [ ] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:ok_woman:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# dbt_hubspot_source v0.6.3
+## Fixes
+- Fixes a bug in the models `stg_hubspot__engagement_meeting.sql` and `stg_hubspot__engagement_meeting_tmp.sql` where the `fivetran_utils.enabled_vars` macro was referencing the wrong variable (`hubspot_engagement_email_enabled`) from the vars list in the `dbt_project.yml`. Also updates `src_hubspot.yml` variable to `hubspot_engagement_meeting_enabled`. This was preventing users from disabling these enagagement_meeting models in their projects.
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:3]: Can you also provide the link to this PR in the CHANGELOG. This just ensures that if people come to the CHANGELOG, they can view the PR where this change was applied.

```suggestion
- Fixes a bug in the models `stg_hubspot__engagement_meeting.sql` and `stg_hubspot__engagement_meeting_tmp.sql` where the `fivetran_utils.enabled_vars` macro was referencing the wrong variable (`hubspot_engagement_email_enabled`) from the vars list in the `dbt_project.yml`. Also updates `src_hubspot.yml` variable to `hubspot_engagement_meeting_enabled`. This was preventing users from disabling these enagagement_meeting models in their projects. ([#85](https://github.com/fivetran/dbt_hubspot_source/pull/85))
```

Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:3]: Can you also add a Contributor section to make sure the customer (in this case @fivetran-seanodriscoll) is credited. 

See a previous CHANGELOG entry on how we give this credit.

Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:3]: Final note, can you also break the `Also` part as a new bullet. Just to make the CHANGELOG a bit easier to digest as smaller pieces.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-catfritz great work on this customer created PR! Additionally, great work catching the additional updates needed for this bug to be fully addressed! 🙌 

I have a few comments included below that I would like for you to address before approving. Let me know if you have any questions. 

In addition to the comments below, you will want to make sure you regenerate the docs as well. We can do this piece together after the PR is approved.

**Comment by fivetran-catfritz** [comment]
@fivetran-joemarkiewicz Thanks for the feedback! I have updated accordingly.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

**Pull request merged**